sourcegraph/learn
# Add alt for header images on post index and post pages

Resolves this [Asana ticket](https://app.asana.com/0/1200612380814575/1200512359516128/f) to add `alt` text to header images by ensuring the `alt` text is available on the existing records themselves and passed appropriately to the relevant components.

NB: The `alt` text is not a dependency here, so if it is not supplied, it will be set to be empty.

### To Test
- Check out branch and ensure that `alt` text is present on header images on both the list items at `/` and individual `/record-slug`s.
- Remove the `imageAlt` text from a record of your choice, and check that everything still renders correctly, and `alt` text is not present.
